generator.yaml

```diff
@@ -99,8 +119,16 @@ resources:
         from:
           operation: DescribeTrainingJob
           path: SecondaryStatus
+      DebugRuleEvaluationStatuses:
+        is_read_only: true
+        # TODO: a list is not printable, but we should be able to select statuses
```

Comment:
I don't think we should have this as a TODO, there can be multiple debugger jobs per training job

Reply:
What is the TODO here? Lists will never be printable, are you planning on splitting into a number of status elements?